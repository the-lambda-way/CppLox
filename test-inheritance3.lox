class Eclair {
  cook() {
    super.cook();
    print "Pipe full of crème pâtissière.";
  }
}
