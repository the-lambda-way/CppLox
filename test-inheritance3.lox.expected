[line 3] Error at 'super': Can't user 'super' in a class with no superclass.
